### ISSUE TITLE
sm multiples accessibility

### ISSUE THREAD

**Issue by julialedur** [COVID19Tracking/website#610]
as Todd Parker pointed out in this great thread, SVG charts aren’t useful to someone on a screen reader because they are just shapes, not readable data values, so he suggested:

- making the small SVGs “invisible” to a screen reader by adding focusable="false" aria-hidden="true" just to keep them from adding clutter as a person navigates through the grid (also hide toggle button and legend)

- adding a string of text that summarizes what the chart is trying to communicate visually by accessibly hiding it so it’s read by a screen reader but invisible in the layout. 
let's add: "This chart shows cumulative total and positive tests in all US states and territories. New York leads the testing race in both total tests and total tests per capita. When we look at absolute number of tests, New York is followed by California, Florida and Texas. But when normalized for population, the states rank changes significantly. In that scenario, New York is followed by Louisiana, Rhode Island and Massachusets."

**Comment by nselikoff** [comment]
I'll work on this :)

**Comment by nselikoff** [comment]
Per discussion with @julialedur in Slack:

* hide the small multiple svgs and legend to screen readers
* instead of hiding the toggle, add "Sort by:" prior to the toggle, which would also clarify to other users what the toggle does
* this way, the order of the individual state links makes sense to all users
* don't add the replacement description for the group listed in the github issue, because it could be out of date quickly depending on how the data changes

After launching v1, we can improve the accessibility of the AreaChart component so that it benefits all area charts on the dashboard/site. It's instructive to see how HighCharts constructs/annotates its line chart here: https://www.highcharts.com/samples/highcharts/accessibility/accessible-line/. Seems that making these charts accessible will mean introducing some data points and axis labels, which will require more discussion and time.
